utils/scripts/createDB.js

```diff
@@ -1,3 +1,4 @@
 const db = require('../database');
 require('dotenv').config();
-console.log("Created database with errors: \n" + db.createSchema());
\ No newline at end of file
+console.log("Created database with errors: \n" + db.createSchema());
+console.log("Created database with errors: \n" + db.populateDatabase());
```

Comment:
this was intentionally left apart from createSchema, if you want to add it to a script, create a seperate script